doc/api/fs.md

```diff
@@ -741,6 +741,40 @@ Returns an object containing commonly used constants for file system
 operations. The specific constants currently defined are described in
 [FS Constants][].
 
+## fs.copyFile(src, dest[, flags], callback)
```

Comment:
I would love to see a code example in the docs

Reply:
There will be one.